tracker/table.go

```diff
@@ -397,6 +400,41 @@ func (e *Order) Title(x int) (title string) {
 	return
 }
 
+func (m *Order) instrumentListFor(trackIndex int) (int, int, error) {
+	track := m.d.Song.Score.Tracks[trackIndex]
+	firstVoice := m.d.Song.Score.FirstVoiceForTrack(trackIndex)
+	lastVoice := firstVoice + track.NumVoices - 1
+	firstIndex, err1 := m.d.Song.Patch.InstrumentForVoice(firstVoice)
+	if err1 != nil {
+		return trackIndex, trackIndex, err1
+	}
+	lastIndex, err2 := m.d.Song.Patch.InstrumentForVoice(lastVoice)
+	if err2 != nil {
+		return trackIndex, trackIndex, err2
+	}
+	return firstIndex, lastIndex, nil
+}
+
+func (m *Order) TrackIndicesForCurrentInstrument() []int {
+	currentTrack := m.d.Cursor.Track
+	return m.d.Song.AllTracksWithSameInstrument(currentTrack)
+}
+
+func (m *Order) CountNextTracksForCurrentInstrument() int {
+	currentTrack := m.d.Cursor.Track
+	count := 0
+	for _, t := range m.TrackIndicesForCurrentInstrument() {
+		if t > currentTrack {
+			count++
+		}
+	}
+	return count
+}
+
+func (m *Order) CurrentTrack() int {
```

Comment:
Is this really needed? `order.CurrentTrack()` vs. `order.Cursor().X` ?

Reply:
no. I don't recall why I put that there